File: Solve/A1/A1-005/A1-005.c
#include <stdio.h>
int m,d;
int main(){
    scanf("%d %d",&m,&d);
    if (d >= 21){
        m = (m+1 > 12) ? 1 : m+1;
    }
    if (m >= 10){
        printf("fall");
    }
    else if (m >= 7){
        printf("summer");
    }
    else if (m >= 4){
        printf("spring");
    }
    else printf("winter");
}

File: Solve/A1/A1-005/A1-005.py
m=int(input())
d=int(input())
if (d >= 21):
    m+=1
    if(m > 12): m=1
if (m>=10):
    print("fall")
elif (m >= 7):
    print("summer")
elif (m >= 4):
    print("spring")
else: print("winter")